### PR TITLE
Add Rviz scene publisher

### PR DESCRIPTION
**Pull request by iory**
# What is this

RvizScenePublisher plugin can publish ``sensor_msgs/Image.msg`` of rviz.
The difference between `videocapture` plugin is `videocapture` generates a video file, but this plugin publishes `sensor_msgs/Image.msg`, which has a time stamp. So users can record it as a bagfile and synchronize other ros topics. This can be useful for users to make a video because we can know the time of the video using the timestamps.

The following gif is the example scene of rviz. the left image is the image published by this plugin.

rviz-scene-publisher

### CODE REVIEW

File: doc/jsk_rviz_plugins/plugins/rviz_scene_publisher.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+RvizScenePublisher
+==================
+
+What is this?
+-------------
+
+RvizScenePublisher plugin can publish ``sensor_msgs/Image.msg`` of rviz.
+
```
Comment (knorth55) [inline doc/jsk_rviz_plugins/plugins/rviz_scene_publisher.rst:8]: can you add `topic_name` property information in doc?

**Comment by knorth55** [comment]
Other things look good to me

**Comment by iory** [comment]
Thank you for your review.
I modified the document by adding the `Publishing Topics`.
https://github.com/jsk-ros-pkg/jsk_visualization/blob/baa58fc4dcc5ec07a1f2495275c75be0ce520358/doc/jsk_rviz_plugins/plugins/rviz_scene_publisher.rst

**Comment by awesomebytes** [comment]
This is a great idea! Thanks for implementing it!

**Pull request merged**